src/main/java/gregtech/api/unification/ore/OrePrefix.java

```diff
@@ -325,6 +325,41 @@ public class OrePrefix {
     public static final OrePrefix circuit = new OrePrefix("circuit", -1, null, null, ENABLE_UNIFICATION, null);
     public static final OrePrefix component = new OrePrefix("component", -1, null, null, ENABLE_UNIFICATION, null);
 
+    // Nuclear stuff, introduced by Zalgo and Bruberu
+    public static final OrePrefix fuelRod = new OrePrefix("fuelRod", -1, null, MaterialIconType.fuelRod, 0,
+            material -> material.hasProperty(PropertyKey.FISSION_FUEL),
+            mat -> Arrays.asList(I18n.format("metaitem.nuclear.tooltip.radioactive"),
+                    I18n.format("metaitem.nuclear.tooltip.duration",
+                            mat.getProperty(PropertyKey.FISSION_FUEL).getDuration()),
+                    I18n.format("metaitem.nuclear.tooltip.temperature",
+                            mat.getProperty(PropertyKey.FISSION_FUEL).getMaxTemperature()),
+                    I18n.format("metaitem.nuclear.tooltip.cross_sections",
+                            mat.getProperty(PropertyKey.FISSION_FUEL).getFastNeutronFissionCrossSection(),
+                            mat.getProperty(PropertyKey.FISSION_FUEL).getSlowNeutronFissionCrossSection()),
+                    I18n.format(
+                            "metaitem.nuclear.tooltip.neutron_time." +
+                                    mat.getProperty(PropertyKey.FISSION_FUEL).getNeutronGenerationTimeCategory(),
+                            mat.getProperty(PropertyKey.FISSION_FUEL).getNeutronGenerationTime())));
+    public static final OrePrefix fuelRodDepleted = new OrePrefix("fuelRodDepleted", -1, null,
+            MaterialIconType.fuelRodDepleted, 0, material -> material.hasProperty(PropertyKey.FISSION_FUEL),
+            mat -> Collections.singletonList(I18n.format("metaitem.nuclear.tooltip.radioactive")));
```

Comment:
Should the radioactive tooltip be applied using an event on anything we consider radioactive? That way things like U235 dust can also be radioactive instead of specific item forms. I think this may be more desirable.